DNN Platform/Library/Prompt/ConsoleCommand.cs

```diff
@@ -1,18 +1,23 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
+using DotNetNuke.Abstractions;
 using DotNetNuke.Abstractions.Portals;
 using DotNetNuke.Abstractions.Prompt;
 using DotNetNuke.Abstractions.Users;
 using DotNetNuke.Collections;
 using DotNetNuke.Services.Localization;
+using Microsoft.Extensions.DependencyInjection;
 using System.Collections.Generic;
 using System.Reflection;
 
 namespace DotNetNuke.Prompt
 {
     public abstract class ConsoleCommand : IConsoleCommand
     {
+        internal static ISerializationManager SerializationManager =>
+    Common.Globals.DependencyProvider.GetRequiredService<ISerializationManager>();
+
```

Comment:
This should be private otherwise any object in the `DotNetNuke.Library` has access to this reference